docs/services/ml/upload-training-script.md

```diff
@@ -421,7 +443,7 @@ This command submits a training job to the previously uploaded `"MyCustomTrainin
 {{% tab name="with-upload" %}}
 
 ```sh {class="command-line" data-prompt="$"}
-viam train submit custom with-upload --dataset-id=<INSERT DATASET ID> --model-org-id=<INSERT ORG ID> --model-name="MyRegistryModel" --model-type="single_label_classification" --model-version="2" --version="1" --path=<path-to-tar.gz> --script-name="MyCustomTrainingScript"
+viam train submit custom with-upload --dataset-id=<INSERT DATASET ID> --model-org-id=<INSERT ORG ID> --model-name="MyRegistryModel" --model-type="single_label_classification" --model-version="2" --version="1" --path=<path-to-tar.gz> --script-name="mycompany:MyCustomTrainingScript"
 ```
 
 This command uploads a script called `"MyCustomTrainingScript"` to the registry under the specified organization and also submits a training job to that script with the input dataset, which generates a new version of the single-classification ML model `"MyRegistryModel"` and publishes that to the registry.
```

Comment:
is the specified organization "mycompany"? ik it's org id but wondering where the mycompany comes in

Reply:
it's the namespace or org-id I believe